plugins/woocommerce/changelog/53507-dev-53381-remove-recommended-payment-methods-for-react-settings-page

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Remove Payment recommendations block from the settings page if reactify-classic-payments-settings feature flag is on.
```

Comment:
Nitpick: Missing new line.

Reply:
```suggestion
Remove the "recommended payment providers" block from the new Payment settings page (but retain it on the classic settings page).
```

Reply:
Also, double-checking if "dev" is the correct type for this change. It's a user-facing change.

Reply:
Thanks, accepted! for now it's dev - we'll wrangle one item for the release that will cover all our Payment NOX work